ariadne/src/Ariadne/Wallet/Cardano/Kernel.hs

```diff
@@ -30,26 +30,27 @@ module Ariadne.Wallet.Cardano.Kernel (
   , hasPending
   ) where
 
-import           Universum hiding (State)
+import Universum hiding (State, init)
```

Comment:
Please avoid applying `stylish-haskell` in this part of the code, or use the Cardano config. We'd like to make import of upstream changes as seamless as possible.

Reply:
Really? As I've seen, imported code is restyled with stylish-haskell config from ariadne repo. My script does that, at least.

Reply:
You are right, I somehow thought it wasn't. In that case please disregard my comment, and I wonder how this file escaped restyling.